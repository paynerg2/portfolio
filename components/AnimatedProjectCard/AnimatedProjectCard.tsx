import { SlideFade, SlideFadeProps, WrapItem } from '@chakra-ui/react';
import { useInView } from 'react-intersection-observer';
import { Project } from '../../utils/interfaces';
import ProjectCard from '../ProjectCard/ProjectCard';

interface Props {
    project: Project;
    slideFadeProps: Partial<SlideFadeProps>;
}

const AnimatedProjectCard = ({ project, slideFadeProps }: Props) => {
    const [ref, inView] = useInView();
    return (
        <SlideFade key={`${project.name}`} in={inView} {...slideFadeProps}>
            <WrapItem ref={ref}>
                <ProjectCard project={project} />
            </WrapItem>
        </SlideFade>
    );
};

export default AnimatedProjectCard;
