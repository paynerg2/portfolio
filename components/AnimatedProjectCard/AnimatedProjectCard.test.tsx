import { render } from '../../test/test-utils';
import AnimatedProjectCard from '../AnimatedProjectCard/AnimatedProjectCard';
import { projects } from '../../test/testData';

const mock = function () {
    return {
        observe: jest.fn(),
        unobserve: jest.fn(),
        disconnect: jest.fn(),
    };
};
window.IntersectionObserver = jest.fn().mockImplementation(mock);

describe('Animated Project Card componenet', () => {
    it('renders the component', () => {
        const component = render(
            <AnimatedProjectCard project={projects[0]} slideFadeProps={undefined} />
        );
        expect(component).toMatchSnapshot();
    });
});
