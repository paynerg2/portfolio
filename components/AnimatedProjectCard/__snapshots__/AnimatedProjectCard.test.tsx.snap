// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Animated Project Card componenet renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    class="chakra-ui-dark"
  >
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <div
        class="chakra-offset-slide"
        style="opacity: 0; transform: translateX(0px) translateY(8px) translateZ(0);"
      >
        <li
          class="chakra-wrap__listitem css-1yp4ln"
        >
          <div
            class="chakra-stack css-1dkkbxz"
          >
            <div
              class="css-e8lz2o"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt="Trackr-preview"
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
              <div
                class="chakra-stack css-114gl1q"
              >
                <h2
                  class="chakra-heading css-15a3bhq"
                >
                  Trackr
                </h2>
                <div
                  class="css-xi606m"
                >
                  Job hunting assistant for aspiring software developers. Tracks applications, interviews, and more. Automatically generates charts to visualize the process.
                </div>
                <div
                  class="css-1dlcyqs"
                >
                  <span
                    class="css-vbl054"
                  >
                    React
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Redux Toolkit
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Typescript
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Styled Components
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Framer Motion
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Nivo
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    NodeJS
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Express
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    MongoDB
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Jest
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Netlify
                  </span>
                  <span
                    class="css-vbl054"
                  >
                    Heroku
                  </span>
                </div>
              </div>
              <div
                class="css-1rjhvx7"
              >
                <div
                  class="css-1ydtbm0"
                >
                  <a
                    class="chakra-button css-1l9myq0"
                    data-testid="github_link"
                    href="https://github.com/paynerg2/trackr"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    See Code
                    <span
                      class="chakra-button__icon css-1hzyiq5"
                    >
                      <svg
                        aria-hidden="true"
                        class="chakra-icon css-13otjrl"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        stroke="currentColor"
                        stroke-width="0"
                        viewBox="0 0 496 512"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                        />
                      </svg>
                    </span>
                  </a>
                  <a
                    class="chakra-button css-1l9myq0"
                    data-testid="website_link"
                    href="https://devjob-trackr.com"
                    target="_blank"
                  >
                    View Site
                    <span
                      class="chakra-button__icon css-1hzyiq5"
                    >
                      <svg
                        aria-hidden="true"
                        class="chakra-icon css-onkibi"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <g
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-width="2"
                        >
                          <path
                            d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                          />
                          <path
                            d="M15 3h6v6"
                          />
                          <path
                            d="M10 14L21 3"
                          />
                        </g>
                      </svg>
                    </span>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="chakra-stack css-od2wl9"
            >
              <a
                class="chakra-button css-1l9myq0"
                data-testid="github_link"
                href="https://github.com/paynerg2/trackr"
                rel="noopener noreferrer"
                target="_blank"
              >
                See Code
                <span
                  class="chakra-button__icon css-1hzyiq5"
                >
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-13otjrl"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 496 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                    />
                  </svg>
                </span>
              </a>
              <a
                class="chakra-button css-1l9myq0"
                data-testid="website_link"
                href="https://devjob-trackr.com"
                target="_blank"
              >
                View Site
                <span
                  class="chakra-button__icon css-1hzyiq5"
                >
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-onkibi"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <g
                      fill="none"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-width="2"
                    >
                      <path
                        d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                      />
                      <path
                        d="M15 3h6v6"
                      />
                      <path
                        d="M10 14L21 3"
                      />
                    </g>
                  </svg>
                </span>
              </a>
            </div>
          </div>
        </li>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="chakra-offset-slide"
      style="opacity: 0; transform: translateX(0px) translateY(8px) translateZ(0);"
    >
      <li
        class="chakra-wrap__listitem css-1yp4ln"
      >
        <div
          class="chakra-stack css-1dkkbxz"
        >
          <div
            class="css-e8lz2o"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="Trackr-preview"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
            <div
              class="chakra-stack css-114gl1q"
            >
              <h2
                class="chakra-heading css-15a3bhq"
              >
                Trackr
              </h2>
              <div
                class="css-xi606m"
              >
                Job hunting assistant for aspiring software developers. Tracks applications, interviews, and more. Automatically generates charts to visualize the process.
              </div>
              <div
                class="css-1dlcyqs"
              >
                <span
                  class="css-vbl054"
                >
                  React
                </span>
                <span
                  class="css-vbl054"
                >
                  Redux Toolkit
                </span>
                <span
                  class="css-vbl054"
                >
                  Typescript
                </span>
                <span
                  class="css-vbl054"
                >
                  Styled Components
                </span>
                <span
                  class="css-vbl054"
                >
                  Framer Motion
                </span>
                <span
                  class="css-vbl054"
                >
                  Nivo
                </span>
                <span
                  class="css-vbl054"
                >
                  NodeJS
                </span>
                <span
                  class="css-vbl054"
                >
                  Express
                </span>
                <span
                  class="css-vbl054"
                >
                  MongoDB
                </span>
                <span
                  class="css-vbl054"
                >
                  Jest
                </span>
                <span
                  class="css-vbl054"
                >
                  Netlify
                </span>
                <span
                  class="css-vbl054"
                >
                  Heroku
                </span>
              </div>
            </div>
            <div
              class="css-1rjhvx7"
            >
              <div
                class="css-1ydtbm0"
              >
                <a
                  class="chakra-button css-1l9myq0"
                  data-testid="github_link"
                  href="https://github.com/paynerg2/trackr"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  See Code
                  <span
                    class="chakra-button__icon css-1hzyiq5"
                  >
                    <svg
                      aria-hidden="true"
                      class="chakra-icon css-13otjrl"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 496 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                      />
                    </svg>
                  </span>
                </a>
                <a
                  class="chakra-button css-1l9myq0"
                  data-testid="website_link"
                  href="https://devjob-trackr.com"
                  target="_blank"
                >
                  View Site
                  <span
                    class="chakra-button__icon css-1hzyiq5"
                  >
                    <svg
                      aria-hidden="true"
                      class="chakra-icon css-onkibi"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <g
                        fill="none"
                        stroke="currentColor"
                        stroke-linecap="round"
                        stroke-width="2"
                      >
                        <path
                          d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                        />
                        <path
                          d="M15 3h6v6"
                        />
                        <path
                          d="M10 14L21 3"
                        />
                      </g>
                    </svg>
                  </span>
                </a>
              </div>
            </div>
          </div>
          <div
            class="chakra-stack css-od2wl9"
          >
            <a
              class="chakra-button css-1l9myq0"
              data-testid="github_link"
              href="https://github.com/paynerg2/trackr"
              rel="noopener noreferrer"
              target="_blank"
            >
              See Code
              <span
                class="chakra-button__icon css-1hzyiq5"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-13otjrl"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 496 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                  />
                </svg>
              </span>
            </a>
            <a
              class="chakra-button css-1l9myq0"
              data-testid="website_link"
              href="https://devjob-trackr.com"
              target="_blank"
            >
              View Site
              <span
                class="chakra-button__icon css-1hzyiq5"
              >
                <svg
                  aria-hidden="true"
                  class="chakra-icon css-onkibi"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                    />
                    <path
                      d="M15 3h6v6"
                    />
                    <path
                      d="M10 14L21 3"
                    />
                  </g>
                </svg>
              </span>
            </a>
          </div>
        </div>
      </li>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
